security/5.0/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Jakarta Security 5.0 (under development)"
+date: 2025-04-15
+summary: "Release for Jakarta EE 11"
+---
+Jakarta Security defines a standard for creating secure Jakarta EE applications in modern application paradigms.
+
+The major theme of this release is absorbing the separate specifications Jakarta Authentication and Jakarta Authorization into Jakarta Security as sub-specs (for example, as JPA once was to EJB).
+
+Whether Jakarta Authentication and Jakarta Authorization will remain available as sub-specs or even whether they will keep their current package name is up to discussion for a release beyond Jakarta Security 5.0. Being lower level SPIs, not directly targeted at the end user, we could allow us some freedom here, but we do have to keep the deprecation and pruning rules of the Jakarta EE platform into account.
+
+Permission stores is a big ticket feature. This can build on the lower level Policy from Jakarta Authorization and would allow users to add permissions (or their alternative form "security constraints") in a user friendly way.
+
+The release furthermore focuses on evolving the API in various ways, and specifically providing APIs for the authorization theme (including interceptors and the above mentioned abstraction for the permission store). 
+
+### New features, enhancements or additions
+* Add support for permission stores #296
+* @RolesAllowed alternative  #295
+* Add SameSite attribute to cookie value for RememberMe annotation #221
+* Easily adding an interceptor to a build-in CDI bean #200
+* Add authentication mechanism for DIGEST #190
+* Add authentication mechanism for CLIENT-CERT #120 
+* Authentication mechanism per URL SECURIY #86 
```

Comment:
Is the last word supposed to be `SECURITY` or `pattern` (like the ticket title)?